### PR TITLE
Add share links to brexit checker results

### PR DESCRIPTION
**Pull request by huwd**
Trello card: https://trello.com/c/fB9KKRrJ/242-add-share-your-results-component

---

Adds social share links that pass on your results URL onto various social media platforms.

## Desktop
Screenshot_2019-12-20 Brexit check what you need to do if there is no deal(2)

## Intermediate
Screenshot_2019-12-20 Brexit check what you need to do if there is no deal

## Tiny Mobile
Screenshot_2019-12-20 Brexit check what you need to do if there is no deal(1)

### CODE REVIEW

File: app/views/brexit_checker/_share_links.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (vanitabarrett) [inline app/views/brexit_checker/_share_links.html.erb]: Extra new line

Comment (huwd) [reply app/views/brexit_checker/_share_links.html.erb]: atch! i _really_ need to get proper linting set up in ruby.

Thanks for spotting.

File: app/helpers/brexit_checker_helper.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 require "addressable/uri"
 
 module BrexitCheckerHelper
+  def encoded_results_url
```
Comment (huwd) [inline app/helpers/brexit_checker_helper.rb]: no test for this, but it's a fairly simple alias.
I think I'd need to stub the request to unit test that here?

**Comment by huwd** [comment]
🤔 can't work out what's going on with the e2e tests here, everything runs fine on local... and I've re-run em.

**Comment by huwd** [comment]
seems to be: 
```
13:46:00  Failed examples:
13:46:00  
13:46:00  rspec ./spec/publisher/publishing_content_to_government_frontend_spec.rb:7 # Publishing content from Publisher to Government Frontend Publishing a Help guide
13:46:00  
13:46:00  Finished in 30.76 seconds (files took 1.93 seconds to load)
13:46:00  1 example, 0 failures
```

Unrelated to this change?

**Comment by huwd** [comment]
oh weird, that change went away...

**Pull request merged**